lab1/README.md

```diff
@@ -403,8 +403,9 @@ func task2() {
 
 ## Exercise
 - What is atomic operation?
-- Will the following work in parallel code?
+- Does the "Bad code" have concurrency issues? Why? And will the following "good code" work in parallel? 
```

Comment:
- Move the good code to the answer section and explain why it works
- Change `Does the "Bad code" have concurrency issues? Why? And will the following "good code" work in parallel?` to `Does the example below have concurrency issues? Why?`

Reply:
OK!